packages/agreement-process/src/model/domain/toEvent.ts

```diff
@@ -102,7 +101,7 @@ export function toCreateEventAgreementAdded(
 export function toCreateEventAgreementUpdated(
   agreement: Agreement,
   version: number
-): CreateEvent<AgreementUpdateEvent> {
+): CreateEvent<AgreementEvent> {
```

Comment:
honestly I find the previous version a bit better 😄 
It is a little safer, and the signature contains more details about what the function does